seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -1158,6 +1158,35 @@
       "translations": {}
     },
     {
+      "id": "587d781d367417b2b2517d0a",
+      "title": "Adjust the Text Format of an Anchor Tag",
```

Comment:
IMHO, this should be `Text Styling` or `Text Decoration` with my personal preference for the later.